.github/CODEOWNERS

```diff
@@ -25,7 +25,8 @@
 # PRLabel: %Storage
 /sdk/storage/                                                        @amishra-dev @zezha-msft @annatisch @rakshith91 @xiafu-msft @tasherif-msft @kasobol-msft
 
-/sdk/applicationinsights/                                            @alexeldeib
+/sdk/applicationinsights/azure-applicationinsights/                  @divya-jay @geneh @alongafni
+/sdk/loganalytics/azure-loganalytics/                                @divya-jay @geneh @alongafni
```

Comment:
Recommend you take the entirety of ```/sdk/applicationinsights``` and ```/sdk/loganalytics``` since there may be changes at that level that impact you.

Reply:
AFAIK other folders are owned by different teams.